src/compiler/compile/Component.ts

```diff
@@ -143,8 +144,16 @@ export default class Component {
 					.replace(process.cwd(), '')
 					.replace(regex_leading_directory_separator, '')
 				: compile_options.filename);
-		this.locate = getLocator(this.source, { offsetLine: 1 });
-
+		const locator = getLocator(this.source, { offsetLine: 1 });
```

Comment:
i tried removing `offsetLine: 1` here but it produces compile errors when used without. Not sure what that offset actually accounts for. Anyone?

Reply:
`originalPositionFor` is 1-based, so it makes sense to add a line here for compatibility since you will need to pass in a 1-based location. could add a comment explaining
```suggestion
		// originalPositionFor is 1-based and source maps are 0-based, so offset by 1 line
		const locator = getLocator(this.source, { offsetLine: 1 });
```

Reply:
originalPositionFor is added in this PR, the offsetLine existed even before. Still not sure why.

Reply:
it's probably because this is being printed to the user as part of the stack trace frame and that's 1-based